src/Normalizers/Normalized/NormalizedModel.php

```diff
@@ -38,10 +39,6 @@ public function getProperty(string $name, DataProperty $dataProperty): mixed
 
     protected function fetchNewProperty(string $name, DataProperty $dataProperty): mixed
     {
-        if ($this->hasModelAttribute($name)) {
-            return $this->properties[$name] = $this->model->getAttributeValue($name);
-        }
-
```

Comment:
Since we remove this piece of code the whole method on the bottom won't be called anymore so can it be removed?